storage/libradb/src/ledger_store/mod.rs

```diff
@@ -71,10 +71,15 @@ impl LedgerStore {
             .iter::<EpochByVersionSchema>(ReadOptions::default())?;
         // Search for the end of the previous epoch.
         iter.seek_for_prev(&version)?;
-        let (epoch_end_version, epoch) = iter
-            .next()
-            .transpose()?
-            .ok_or_else(|| LibraDbError::NotFound(format!("version {}", version)))?;
+        let (epoch_end_version, epoch) = match iter.next().transpose()? {
+            Some(x) => x,
+            None => {
+                // There should be a genesis LedgerInfo at version 0 (genesis only consists of one
+                // transaction), so this normally doesn't happen. However this part of
+                // implementation doesn't need to rely on this assumption.
+                return Ok(0);
```

Comment:
I feel the previous way is more explicit, if there's no genesis ledger info, storage should return an Error instead of implicit 0

Reply:
The previous way was more explicit in the sense that if the genesis LI is missing due to db corruption, the code would return an error instead of an incorrect result. However this protection is limited to the situation where only the genesis LI is missing and not very general. On the other hand, if we want to use the previous code, we will need to change the code in `types/src/proptest_types.rs` to make sure the first block is always size 1 in the test input. So I don't feel it's worth the effort...

Reply:
I probably misunderstood `NotFound`, as I used in the `get_epoch_ledger_info` I didn't realize it's only meant for db corruption.
I thought it's just used to indicate that the caller is trying to find sth not exist

Reply:
Yeah, if things are okay no error should happen. The `NotFound` error is used to when something that is supposed to exist cannot be found.